### ISSUE TITLE
Only animates on initialColor

### ISSUE THREAD

**Issue by ustincameron** [joeyschroeder/react-native-animated-background-color-view#1]
I'm using `<AnimatedBackgroundColorView
                    color={this.state.bgColor}
                    initialColor='#000'
                    duration={6000}
                    delay={1000}
                >` and it updates the background whenever I update state, however it's never animated. Have also tried updating the var within render() but same results. Any ideas?

**Comment by joeyschroeder** [comment]
Hey @socca1157, could you send me a larger code sample? I'm not sure I'm able to diagnose the issue from what you described.

In the meantime, here's a short example that I've tested that works.  It sets the background initially to white, then delays 1000ms and transitions to blue, then at 4000ms after mount, `this.updateColor` is fired which sets the color to red.
```
import React, { Component } from 'react';
import { StyleSheet, Text } from 'react-native';

import { AnimatedBackgroundColorView } from 'react-native-animated-background-color-view';

const styles = StyleSheet.create({
  root: {
    alignItems: 'center',
    flex: 1,
    justifyContent: 'center'
  }
});

export class App extends Component {
  constructor() {
    super();
    this.state = { color: '#00aced' };

    this.updateColor = this.updateColor.bind(this);
  }

  componentDidMount() {
    setTimeout(this.updateColor, 4000);
  }

  updateColor() {
    this.setState({ color: '#ff0054' });
  }

  render() {
    const { color } = this.state;

    return (
      <AnimatedBackgroundColorView
        color={color}
        delay={1000}
        initialColor="#fff"
        style={styles.root}
      >
        <Text>Test</Text>
      </AnimatedBackgroundColorView>
    );
  }
}
```

**Comment by joeyschroeder** [comment]
Closed due to inactivity.

**Comment by MossP** [comment]
For the record, I see the same issue. Code sample attached.
I'll update if I find out why.

```
import React, { Component } from 'react';
import { View } from 'react-native';
import { styles } from './styles';
import  * as AppShape from '../../../../shared/redux/base/types';
import { AnimatedBackgroundColorView } from 'react-native-animated-background-color-view';

interface Props {
  wrapState: AppShape.AppWrapperState;
}

export class ComponentBody extends Component<Props>{
  constructor(props:Props) {
    super(props);
  }

  render() {
    return (
      <AnimatedBackgroundColorView
        style={[styles.root]}
        color={this.props.wrapState.backgroundColor}
        // initialColor={'#333333'}
        duration={5000}>
        {this.props.children}
      </AnimatedBackgroundColorView>
    );
  }
}
```

**Comment by MossP** [comment]
Strangely. Bringing up the RN Dev menu and pressing cancel makes this work (one more time) then the same behaviour occurs again.

**Comment by joeyschroeder** [comment]
This issue will be resolved when #2 is resolved.
